articles/quickstarts/index.md

```diff
@@ -19,17 +19,150 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
+|**Q# and .NET (C#, F#)**|[Install](#configure-for-vs-code) |[Install](#configure-for-visual-studio)|&#10006; |[Install](#configure-for-visual-studio) |
+
+### Configure for VS Code
+
+#### Prerequisites 
+
+- [.NET Core SDK 3.1 or later](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+
+To configure VS Code:
+
+1. Download and install [VS Code](https://code.visualstudio.com/download) (Windows, Linux and macOS).
+2. Install the [Microsoft QDK for VS Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode).
+
+To start writing Q# code with VS Code, see [Develop with Q# applications](xref:microsoft.quantum.install.standalone). 
+
+### Configure for Visual Studio
+
+#### Prerequisites 
+
+- [.NET Core SDK 3.1 or later](https://www.microsoft.com/net/download)
+
+To configure Visual Studio:
+
+1. Download and install [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3 or greater, with the .NET Core cross-platform development workload enabled.
+2. Download and install the [Microsoft QDK extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).
+
+To start writing Q# code with Visual Studio, see [Develop with Q# applications](xref:microsoft.quantum.install.standalone).
+
+### Configure for Jupyter Notebooks and Python
+
+Jupyter Notebooks allow running in-place code alongside instructions, notes, and other content. This environment is ideal for writing Q# code with embedded explanations or quantum computing interactive tutorials. 
+
+IQ# (pronounced i-q-sharp) is an extension to the .NET Core SDK, primarily used by Jupyter and Python, that provides the core functionality for compiling and simulating Q# operations.
+
+Here's what you need to do to start creating your own Q# notebooks.
+
+### [Install using conda (recommended)](#tab/tabid-conda)
+
+1. Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/products/individual#Downloads). **Note:** 64-bit installation required.
+
+1. On Windows, open an Anaconda Prompt, or on macOS or Linux, open a terminal window.
+   - Or, if you prefer to use PowerShell or pwsh: open a shell, run `conda init powershell`, then close and re-open the shell.
+1. Create and activate a new conda environment named `qsharp-env` with the required packages (including Jupyter Notebook and IQ#) by running the following commands:
+    ```
+    conda create -n qsharp-env -c quantum-engineering qsharp notebook
+    conda activate qsharp-env
+    ```
+1. Run `python -c "import qsharp"` from the same terminal to verify your installation and populate your local package cache with all required QDK components.
+
+To start writing Q# code with Python, see [Develop with Q# and Python](xref:microsoft.quantum.install.python).
```

Comment:
I think this part can be confusing for new users. Currently, Q# Jupyter notebooks only allow to run Q# code and it cannot be combined with Python cells. Many people tend to make the link Jupyter=Python, but this is not the case. Although you can use Python Jupyter notebooks as host programs to call `.qs` files. I would propose to remove Python to avoid confusion and call them "Q# Jupyter notebooks" to make clearer the difference. 

Any thoughts @rmshaffer @cgranade and @KittyYeungQ?

Reply:
Agreed. I like the fact that we currently have separate installation pages for Q# Jupyter Notebooks and Q# + Python. It reinforces that these are different experiences (even though, yes, the installation steps are identical).

Reply:
(Apologies for a slight tangent.)

> Many people tend to make the link Jupyter=Python

We've definitely seen some of that confusion, I agree, but it's also the case that Jupyter was designed from the start to be a multilanguage platform (indeed, the name derives from a portmanteau of Julia, Python, and R). Perhaps we can mitigate some of that misunderstanding by always being explicit about what language a given notebook is in when we link to it.